crowdsec-docs/docs/concepts.md

```diff
@@ -7,202 +7,55 @@ sidebar_position: 1
 
 # Global overview
 
-The Security Engine runtime revolves around a few simple concepts:
+# Log Processor (LP)
 
- - It reads logs (defined via [datasources](/data_sources/introduction.md) configuration)
- - Those logs are parsed via [parsers](/parsers/introduction.mdx) and eventually [enriched](/parsers/enricher.md)
- - Those normalized logs are matched against the [scenarios](/scenarios/introduction.mdx) that the user has deployed
- - When a scenario is "triggered", CrowdSec generates an [alert](/concepts.md#alerts) and eventually one or more associated [decisions](/concepts.md#decisions):
-    - The alert is here mostly for traceability and will stay even after the decision expires
-    - The decision, on the other hand, is short-lived and tells *what* action should be taken against the offending IP/range/user...
- - This information (the signal, the associated decisions) is then sent to [Local API](/local_api/intro.md) and stored in the database
+> The Log Processor is in charge of the detection of bad behaviors, based on your logs or your HTTP trafic.
```

Comment:
I'd precise:
The log processor is part of CrowdSec Security Engine, it is in charge of...